### PR TITLE
docs: updating the README and Contributing Guide

### LINKED ISSUE #2205: docs: Updates needed
### Issue Description

Here are two issues that I noticed in the docs:
### README
I noticed that the [Maintainers section does not mention Aftan](https://github.com/rupali-codes/LinksHub#maintainers-). This can cause new contributors to be confused.

### CONTRIBUTING Guide 
While the examples in the steps to adding categories/subcategories are descriptive, it could be improved.

### Screenshots or Examples (if applicable)

_No response_

### Readme

- Add Aftan to the Maintainer’s section 

### Contributing Guide

- Add code snippets that describe the examples. Something similar to the [Appending Links](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#follow-these-steps-to-add-your-links-to-the-hub) section.

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by CBID2**
## Fixes Issue
Closes #2205 

## Changes proposed
This PR addresses the following issues:
1. Adds Aftan to the Maintainer's section in the README file
2. Adds a code snippet in the section of Contributing guide that describes the steps to adding categories/subcategories.

### Change to README file 
code snippet in contributing guide

### Change to Contributing Guide 
aftan's image in the maintainer's section

### CODE REVIEW

**Comment by CBID2** [comment]
@Anmol-Baranwal, @aftabrehan, and @rupali-codes, please review my PR.

File: README.md [hunk 1]
```diff
@@ -112,9 +112,10 @@ Need help while contributing? Tag any of the maintainers when creating the issue
...
+<td align="center" width="200"><pre><a href="https://github.com/rupali-codes"><img src="https://avatars.githubusercontent.com/u/78981177?v=4" width="200" alt="GitHub Profile picture of Rupali Haldiya" /><br><sub>Rupali Haldiya</sub></a><br>@rupali-codes</pre></td>
+<td align="center" width="200"><pre><a href="https://github.com/CBID2"><img src="https://avatars.githubusercontent.com/u/105683440?v=4" width="200" alt="GitHub Profile picture of Christine Belzie" /><br><sub>Christine Belzie</sub></a><br>@CBID2</pre></td>
+<td align="center" width="200"><pre><a href="https://github.com/Anmol-Baranwal"><img src="https://avatars.githubusercontent.com/u/74038190?v=4" width="200" alt="GitHub Profile picture of Anmol Baranwal" /><br><sub>Anmol Baranwal</sub></a><br>@Anmol-Baranwal</pre></td>
+<td align="center" width="200"><pre><a href="https://github.com/aftabrehan"><img src="https://avatars.githubusercontent.com/u/93012310?v=4" width="200" alt="GitHub Profile Picture of Aftan Brehan"/><br><sub>Aftan Brehan</sub></a><br>@aftabrehan</pre></td>
```
Comment (aftabrehan) [inline README.md:118]:
```suggestion
<td align="center" width="200"><pre><a href="https://github.com/aftabrehan"><img src="https://avatars.githubusercontent.com/u/93012310?v=4" width="200" alt="GitHub Profile Picture of Aftab Rehan"/><br><sub>Aftab Rehan</sub></a><br>@aftabrehan</pre></td>
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -135,7 +135,15 @@ Follow these steps to setup LinksHub on your local machine
...
-  - You can check out similar examples [here](components/TopBar/CategoryDescriptions.ts). It's essential to add a description when submitting a pull request to add a subcategory; to merge it in the codebase.
+  - Here's how it looks altogether:
+
+  ```ts
+  //category
+   subcategory name: 
+   `description of subcategory`,
+   ```
+
+  **Note:** It's essential to add a description when submitting a pull request to add a subcategory to merge it in the codebase.
```
Comment (aftabrehan) [inline CONTRIBUTING.md:146]:
```suggestion
> [!NOTE]  
> It's essential to add a description when submitting a pull request to add a subcategory to merge it in the codebase.
```

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Excellent PR, @CBID2 💪 🚀 💯 

Could you please check these suggestions?

**Comment by CBID2** [comment]
> Excellent PR, @CBID2 💪 🚀 💯 
> 
> 
> 
> Could you please check these suggestions?

Done @aftabrehan

**Review by Anmol-Baranwal — COMMENTED** [review]
You removed the context where they can check category descriptions. Will anyone who is new to the codebase not face a problem finding where they can check the code, since it is in another component `TopBar`?

It's up to you. Let me know, and I will approve the PR.

image

**Comment by CBID2** [comment]
> You removed the context where they can check category descriptions. Will anyone who is new to the codebase not face a problem finding where they can check the code, since it is in another component `TopBar`?
> 
> 
> 
> It's up to you. Let me know, and I will approve the PR.
> 
> 
> 
> image
> 
> 

Good point @Anmol-Baranwal! :) I'll add that back in there.

**Review by aftabrehan — APPROVED** [review]

**Review by rupali-codes — APPROVED** [review]
Lgtm

**Comment by CBID2** [comment]
> You removed the context where they can check category descriptions. Will anyone who is new to the codebase not face a problem finding where they can check the code, since it is in another component `TopBar`?
> 
> 
> 
> It's up to you. Let me know, and I will approve the PR.
> 
> 
> 
> image
> 
> 

Made the change @Anmol-Baranwal! 😁

**Comment by Anmol-Baranwal** [comment]
> @Anmol-Baranwal, my PR got merged before I could make the change. Can you add it in a new PR? :)

No worries! If anyone faces that problem, we can let them know.

No need of a new PR.

**Comment by CBID2** [comment]
> > @Anmol-Baranwal, my PR got merged before I could make the change. Can you add it in a new PR? :)
> 
> 
> 
> No worries! If anyone faces that problem, we can let them know.
> 
> 
> 
> No need of a new PR.

Ok good @Anmol-Baranwal! 😁

**Pull request merged**